templates/web/base/dashboard/index.html

```diff
@@ -143,6 +143,14 @@
       [% END %]
     [% END %]
 </select>
+</p>
+<p>
+<label for="start_date">Start Date</label><input name="start_date" type="date" value="[% start_date %]">
```

Comment:
While I'm here, there's also a superfluous `</select>` above the Report state field.